toolkit/scripts/toolchain.mk

```diff
@@ -130,7 +137,7 @@ $(raw_toolchain): $(toolchain_files)
 # Output:
 # out/toolchain/built_rpms
 # out/toolchain/toolchain_built_rpms.tar.gz
-$(final_toolchain): $(raw_toolchain) $(BUILD_SRPMS_DIR)
+$(final_toolchain): $(raw_toolchain) $(STATUS_FLAGS_DIR)/build_toolchain_srpms.flag
```

Comment:
Per comment in the SRPM makefile, I think we might want to have `$(BUILD_TOOLCHAIN_SRPMS_DIR)` here instead?

Reply:
Resolved in the other comment thread, the srpms ultimately share the same directory.